extensions/redis-client/runtime/src/test/java/io/quarkus/redis/runtime/client/VertxRedisClientFactoryTest.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.redis.runtime.client;
+
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+
+class VertxRedisClientFactoryTest {
+
+    @Test
+    void shouldApplyQuery() {
+
+        String applied = VertxRedisClientFactory.applyClientQueryParam(
+                "quarkus-app", URI.create("redis://localhost:6379")
+        );
+
+        SoftAssertions.assertSoftly(softly -> {
```

Comment:
Not sure why the soft assertions here? The other tests don't use them. Also, asserting that `applied` is not blank is futile when you next assert that it is equal to a given string.